tests/storage/utils.py

```diff
@@ -238,7 +240,7 @@ def set_permissions(
             role_ref_kind=cluster_role.kind,
             role_ref_name=cluster_role.name,
         ) as role_binding:
-            yield [cluster_role, role_binding]
+            yield cluster_role, role_binding
```

Comment:
No need to change this since we don’t use the return values from `set_permissions`, but if you're updating this anyway, an empty yield might make the intent clearer